autogpt_platform/backend/backend/blocks/stripe_link/spend_request_test.py

```
@@ -0,0 +1,257 @@
+    totals = [{"type": "total", "display_text": "Total", "amount": 12000}]
+    body = await run_create(line_items=items, totals=totals, metadata={"order": "1"})
+
+    assert body["line_items"] == items
+    assert body["totals"] == totals
+    assert body["metadata"] == {"order": "1"}
+
+
+@pytest.mark.asyncio
+async def test_context_below_the_minimum_is_rejected():
+    """This string is what the user reads before approving a charge."""
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)